episodes/02-filedir.md

```diff
@@ -821,6 +821,23 @@ Library/              Users/
 Network/              Volumes/
 ```
 
+
+:::::::::::::::::::::::::::::::::::::::::  callout
+### When to use short or long options
+Sometimes options exist as both short and long options.
+In this case there are no rules about which to choose, but there is a good
+practice rule-of-thumb:
+
+- Use the short option when typing commands directly into bash. 
+  This helps to minimise typing and enable you to get your task done faster.
+- Use the long option when using commands in a bash script.
+  Long options are preferred in this situation because you will probably end up
+  writing the command only once but you, or other people, may need to read it many times! 
+  The long option is a better choice in this situation because it provides
+  greater clarity about the option being used.
+  
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
```

Comment:
This is nice, but quite long in a time constrained lesson. What about:

```suggestion
:::::::::::::::::::::::::::::::::::::::::  callout
### When to use short or long options
When options exist as both short and long options:

- Use the short option when typing commands directly into bash 
  to minimize typing. Enable you to get your task done faster.
- Use the long option in scripts to provide clarity. It will be read
  many times and typed once.
  
::::::::::::::::::::::::::::::::::::::::::::::::::

```
Feel free to modify

Reply:
I'm happy with that. I reworded very slightly. You wouldn't be the first person to imply (or even outright say) that I'm a bit verbose!

Reply:
Many thanks. Made a few more changes. Hope ok. Feel free to make a new pull request if further changes needed.